.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!--
+Fill in the relevant information below to help triage your issue.
+Pick the target branch based on the following criteria:
+  * Documentation improvement: master branch
+  * Bugfix: master branch
+  * QA improvement (additional tests, CS fixes, etc.) that does not change code
+    behavior: master branch
+  * New feature or refactor or existing code: develop branch
```

Comment:
This is also from template, but there is something wrong - `New feature or refactor or existing code`.

BTW Do we want develop for refactor that does not change the code behaviour?

Reply:
It should read "New feature, or refactor of existing code".

Marco would definitely argue that a refactor is considered a new _feature_, particularly as it can potentially introduce BC breaks or be used to allow new features to evolve.